src/inferenceql/query/literal.cljc

```diff
@@ -14,12 +15,14 @@
                       node)]
     [[:value child]] (read child)
 
-    [[:bool s]]          (edn/read-string s)
-    [[:float s]]         (edn/read-string s)
-    [[:int s]]           (edn/read-string s)
-    [[:nat s]]           (edn/read-string s)
-    [[:simple-symbol s]] (edn/read-string s)
-    [[:string s]]        (edn/read-string (str \" s \"))
+    [[:bool s]]             (edn/read-string s)
+    [[:float s]]            (edn/read-string s)
+    [[:int s]]              (edn/read-string s)
+    [[:nat s]]              (edn/read-string s)
+    [[:identifier child]]   (read child)
+    [[:simple-symbol s]]    (edn/read-string s)
+    [[:delimited-symbol s]] (edn/read-string (q.string/safe-symbol s))
+    [[:string s]]           (edn/read-string (str \" s \"))
```

Comment:
@zane Should this be unaligned? I believe it was already aligned before, but I can remove that.

Reply:
Works for me.